src/components/custom/dashboard/shared/chat/RealTimeChatInterface.jsx

```
@@ -0,0 +1,914 @@
+        <div className="mt-1 flex items-center justify-between">
+          <p className="max-w-[150px] truncate text-sm text-gray-400">
+            {conversation.isBlocked 
+              ? "Blocked chat" 
+              : conversation.isNewChat 
+                ? "Start conversation" 
+                : "Chat active"}
+          </p>
+          <div className="flex items-center gap-2">
+            {conversation.isBlocked && (
+              <span className="flex h-5 w-5 items-center justify-center rounded-full bg-orange-500">
+                <svg className="h-3 w-3 text-white" fill="none" stroke="currentColor" viewBox="0 0 24 24">
```

Biome (2.1.2)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)